services/static-webserver/client/source/class/osparc/WindowSizeTracker.js

```diff
@@ -48,8 +48,8 @@ qx.Class.define("osparc.WindowSizeTracker", {
   },
 
   statics: {
-    WIDTH_BREAKPOINT: 1280, // HD 1280x720
-    HEIGHT_BREAKPOINT: 720, // HD 1280x720
+    WIDTH_BREAKPOINT: 1180, // HD 1180x720 - iPad Pro 11" 1194x834 inclusion
+    HEIGHT_BREAKPOINT: 720, // HD 1180x720 - iPad Pro 11" 1194x834 inclusion
```

Comment:
It's not the [720p HD](https://en.wikipedia.org/wiki/720p) anymore then

Reply:
@odeimaiz any reason that would make a difference. We are interested in devices not video? Or am I missing something. I have tested quite a bit and haven't come up with any issues regarding the new min size.

Reply:
It about just about the "HD" text in the comment